pycbc/inference/likelihood.py

```diff
@@ -33,6 +33,7 @@
 from pycbc.types import Array
 from pycbc.io import FieldArray
 import numpy
+from pycbc.inference.recal import Recalibrate
```

Comment:
As I said before, don't create the calibration here. Just pass the calibration instance to the likelihood class.